doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -248,6 +248,9 @@ nRF Desktop
       The public key used by MCUboot for validating the application image is securely stored in the KMU hardware peripheral.
       To simplify the programming procedure, the application is configured to use the automatic KMU provisioning.
       The KMU provisioning is performed by the west runner as a part of the ``west flash`` command when the ``--erase`` or ``--recover`` flag is used.
+    * Application configurations to avoid using the following deprecated Kconfig options: :ref:`CONFIG_DESKTOP_HID_REPORT_EXPIRATION <config_desktop_app_options>` and :ref:`CONFIG_DESKTOP_HID_EVENT_QUEUE_SIZE <config_desktop_app_options>`.
```

Comment:
```suggestion
    * Application configurations to avoid using the deprecated Kconfig options :ref:`CONFIG_DESKTOP_HID_REPORT_EXPIRATION <config_desktop_app_options>` and :ref:`CONFIG_DESKTOP_HID_EVENT_QUEUE_SIZE <config_desktop_app_options>`.
```